examples/pose-apriltag/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+project(rs-pose-apriltag)
+
+if(BUILD_EXAMPLES)
+
+    find_path(APRILTAG_INC NAMES apriltag.h PATH_SUFFIXES apriltag)
+    find_library(APRILTAG_LIB apriltag)
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(apriltag DEFAULT_MSG APRILTAG_INC APRILTAG_LIB)
```

Comment:
In my setup, I installed the `apriltag` library in a custom directory. The only way of making it work was to set `APRILTAG_INC=mypath/install/include/apriltag` and `APRILTAG_LIB=mypath/install/lib/libapriltag.so` in cmake. Note that I had to specify the inner subdirectory in `include/apriltag` and the library file `libapriltag.so`. Setting `CMAKE_PREFIX_PATH` didn't help, or I didn't do it correctly.

Is this behavior expected?

Reply:
@dorian3d, `CMAKE_PREFIX_PATH` should work.   Make sure to kill your build directory if you forgot to set `CMAKE_PREFIX_PATH` the first time, or run `cmake -UAPRILTAG_INC -UAPRILTAG_LIB .` to force the auto-detection to happen again.

Reply:
Starting from scratch helped and `CMAKE_PREFIX_PATH` works 👍